apps/site/lib/site_web/controllers/schedule/finder_api.ex

```diff
@@ -33,6 +38,29 @@ defmodule SiteWeb.ScheduleController.FinderApi do
   # How many seconds a departure is considered recent
   @recent_departure_max_age 600
 
+  def stops_for_route_pattern(conn, %{
+        "route_id" => route_id,
+        "pattern_id" => pattern_id
+      }) do
+    {:ok, route} = LineHelpers.get_route(route_id)
+
+    stops_by_direction =
+      Map.new()
+      |> Map.put("0", get_stops_for_route_pattern(route, 0, pattern_id))
+      |> Map.put("1", get_stops_for_route_pattern(route, 1, pattern_id))
+
+    json(conn, stops_by_direction)
+  end
+
+  defp get_stops_for_route_pattern(route, direction_id, pattern_id) do
+    pattern = RoutePatterns.Repo.get(pattern_id)
+    trip = Schedules.Repo.trip(pattern.representative_trip_id)
+    shape = Routes.Repo.get_shape(trip.shape_id) |> List.first()
```

Comment:
When [we parse route patterns we add the shape_id and stop_ids](https://github.com/mbta/dotcom/blob/74a4703d61005cfbd2f64ea72fa8db35921215f3/apps/route_patterns/lib/route_pattern.ex#L102), couldn't we use those?